io.github.openWMail.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.openwmail.desktop</id>
```

Comment:
I think it should be io.github.openWMail.desktop instead.

Reply:
yes, it should. everything needs to match.

Reply:
ah, no, I've looked closer this needs to be whatever the app is called so currently "io.github.openWMail" however that's not actually a full ID, github ones need four parts. So you could have:

- io.github.openWMail.openWMail
- io.github.openWMail.desktop
- io.github.openWMail.Desktop

etc. but whatever it is, the app-id and the name of the JSON file etc. needs to be updated to match.